packages/cli-kit/src/public/node/api/partners.ts

```diff
@@ -55,6 +57,28 @@ const ScriptServiceProxyQuery = gql`
   }
 `
 
-interface ScriptServiceProxyQuerySchema {
-  scriptServiceProxy: unknown
+interface Deprecation {
+  supportedUntilDate?: string
+}
+
+interface WithDeprecations {
+  deprecations: Deprecation[]
+}
+
+/**
+ * Sets the next deprecation date if response extensions contain deprecation dates.
```

Comment:
could we add a clarification here that `extensions` means "graphql extension"? I found this description useful: https://www.apollographql.com/docs/resources/graphql-glossary/#extensions